src/rust-crypto/verification.ts

```diff
@@ -322,6 +333,10 @@ export class RustVerificationRequest
             throw new Error(`Unsupported verification method ${method}`);
         }
 
```

Comment:
Is this the critical part that causes us to avoid the race? If so, can we have a comment here explaining that?

Reply:
yes, sorry. Have added a comment.